src/main/java/seedu/address/ui/PersonalBestTable.java

```diff
@@ -41,6 +41,7 @@ public PersonalBestTable(ObservableList<Exercise> e) {
         exerciseDate.setCellValueFactory(new PropertyValueFactory<Exercise, String>("exerciseDate"));
         exerciseWeight.setCellValueFactory(new PropertyValueFactory<Exercise, String>("exerciseWeight"));
         exerciseReps.setCellValueFactory(new PropertyValueFactory<Exercise, String>("exerciseReps"));
+        exerciseName.setText("Personal Best Exercise");
```

Comment:
I suggest moving this to fxml for consistency

Reply:
Ok, please kindly do it for me  =D

Reply:
damn